src/main/java/com/cleanroommc/groovyscript/compat/mods/MinecraftModContainer.java

```diff
@@ -0,0 +1,60 @@
+package com.cleanroommc.groovyscript.compat.mods;
+
+import com.cleanroommc.groovyscript.compat.vanilla.VanillaModule;
+import com.google.common.base.Supplier;
+import com.google.common.base.Suppliers;
+import it.unimi.dsi.fastutil.objects.ObjectOpenHashSet;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Set;
+
+public final class MinecraftModContainer extends GroovyContainer<VanillaModule> {
+
+    private final String modId = "minecraft";
+    private final String containerName = "Minecraft";
+    private final Supplier<VanillaModule> modProperty;
+    private final Collection<String> aliases;
+
+    MinecraftModContainer() {
+        this.modProperty = Suppliers.memoize(() -> {
+            VanillaModule t = VanillaModule.INSTANCE;
+            t.addPropertyFieldsOf(t, false);
+            return t;
+        });
+        Set<String> aliasSet = new ObjectOpenHashSet<>();
+        aliasSet.add("mc");
```

Comment:
would `vanilla` be a good alias? would allow `mods.vanilla`, which i find rather amusing.
i believe using `aliasSet.addAll(Alias.generateOf("Vanilla", containerName).and("MC", "mc"));` should work.